### ISSUE TITLE
2. Ontbrekend HTML lang-attribuut (Homepage template A11y)

### ISSUE THREAD

**Issue by TessaViergever** [frameless/developer.overheid.nl#115]
### 👪 Related issue
 [#108](https://github.com/frameless/developer.overheid.nl/issues/108) 

🔗  Aparte [(draft) PR](https://github.com/nl-design-system/rijkshuisstijl-community/pull/1587) gemaakt om oplossing in project te implementeren.

### Betrokken pagina's

[Homepagina template](https://rijkshuisstijl-community-templates-3ix5516vd-nl-design-system.vercel.app/developer-overheid/homepage)

### Beschrijf het probleem

De html lang-attribuut ontbreekt nog steeds.

### Relevante code

```html
`<html class="utrecht-root">`
```

### WCAG-richtlijn

WCAG 3.1.1 Language of Page Level A

### Voeg een afbeelding toe (indien relevant)

![Screenshot of the developer tools which show the page lacks a lang attribute](https://github.com/user-attachments/assets/87c118ac-8746-4caf-a7db-71b72f64b94e)

### To-do's

Je kan de nextjs app (`rhc-templates`folder) aanpassen om een lang attribute te zetten, dat zou correct zijn als je de template bekijkt vanaf die app, maar potentially niet de tijd/moeite waard.
Tussen oplossing zou kunnen zijn: language instellen op hoogste element op de pagina met duidelijk erbij zetten dat dit op html hoort? (bijv als issue of comment in de code)

- [ ] implementeren van language decorator
- [ ] in `/app/layout.tsx` op de `root` component de `lang `en `dir` instellen
